python/whylogs/experimental/core/metrics/udf_metric.py

```diff
@@ -290,25 +340,68 @@ def upper(x):
 
     resolvers: List[ResolverSpec] = list()
     udfs: Dict[str, Callable[[Any], Any]]
-    for col_name, submetrics in _col_name_submetrics[schema_name].items():
+    schema_names = schema_name if isinstance(schema_name, list) else [schema_name]
+    if include_default_schema and "" not in schema_names:
+        schema_names = [""] + schema_names
+
+    col_udfs: Dict[str, List[Tuple[str, Callable[[Any], Any]]]] = defaultdict(list)
+    col_submetric_schema: Dict[str, SubmetricSchema] = dict()
+    col_type_mapper: Dict[str, TypeMapper] = dict()
+
+    type_udfs: Dict[DataType, List[Tuple[str, Callable[[Any], Any]]]] = defaultdict(list)
+    type_submetric_schema: Dict[DataType, SubmetricSchema] = dict()
+    type_type_mapper: Dict[DataType, TypeMapper] = dict()
+
+    for schema_name in schema_names:
+        for col_name, submetrics in _col_name_submetrics[schema_name].items():
+            col_udfs[col_name] += submetrics
+            if col_submetric_schema.get(col_name) and _col_name_submetric_schema[schema_name].get(col_name):
+                logger.warning(f"Multiple submetric schemas registered for column {col_name}")
```

Comment:
We might add hints to the user on what the impact of this is or a hint as to what they should do to correct?